src/objects/GS_Admin_Checklist_Section__mdt.object

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomObject xmlns="http://soap.sforce.com/2006/04/metadata">
+    <description>NPSP Getting Started Page sections</description>
+    <fields>
+        <fullName>Description_Label__c</fullName>
+        <description>The Description of a section item on the Get Started page.</description>
+        <externalId>false</externalId>
+        <fieldManageability>DeveloperControlled</fieldManageability>
+        <inlineHelpText>The Description of a section item on the Get Started page.</inlineHelpText>
+        <label>Description Label</label>
+        <length>255</length>
+        <required>true</required>
+        <type>Text</type>
+        <unique>false</unique>
+    </fields>
+    <fields>
+        <fullName>Position__c</fullName>
+        <description>The position of this section item on the Get Started page. Enter a number greater than zero.</description>
```

Comment:
Since I haven't gotten a clear answer on whether you can enter 0 in this field, please change the description and inlinehelptext to: "The position of this section on the Get Started page."

Reply:
It could be any number. It is used to sort the sections but there is no validation at all.